src/main/java/mate/academy/service/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.service;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String USER = "User";
+    private static final String PASSWORD = "Password";
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/library";
```

Comment:
Storing database credentials directly in the source code is not secure. Consider externalizing these properties to a configuration file or environment variables to enhance security and flexibility.